apps/ui-library/registry/default/blocks/social-auth-react-router/app/routes/login.tsx

```diff
@@ -0,0 +1,63 @@
+import { createClient } from '@/registry/default/clients/react-router/lib/supabase/server'
+import { Button } from '@/registry/default/components/ui/button'
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/registry/default/components/ui/card'
+import { type ActionFunctionArgs, redirect, useFetcher } from 'react-router'
+
+export const action = async ({ request }: ActionFunctionArgs) => {
+  const { supabase } = createClient(request)
+
+  const { data, error } = await supabase.auth.signInWithOAuth({
+    provider: 'github',
+    options: {
+      redirectTo: `${window.location.origin}/auth/oauth?next=/protected`,
```

Comment:
I don't think `window` is available here because it's running server side

Reply:
Fixed.